docs/user-guide/src/tour/testing.md

```diff
@@ -1,8 +1,76 @@
 # Testing
 
-**TODO**
+> **Recommended reading:** 
+> [2.6: Testing](https://rust-unofficial.github.io/too-many-lists/first-test.html),
+
+The linked chapter in the "Learning Rust With Entirely Too Many Linked Lists" tutorial explains how testing normal Rust code works. In this chapter we will show some differences between testing and verification and of the guarantees that each provides.
+
+Note: Normal tests (marked with `#[cfg(test)]` or `#[test]`) are currently ***not*** checked by Prusti, but this may be added in the future. If you remove the `test` markers, Prusti will check any test like it would a normal function.
+
+## Differing guarantees of verification and testing
+
+Static verification has stronger guarantees than testing.
+Running tests is only be possible for a small subset of all possible input values.
+Take as an example a function taking a single value of type `u64`. The range of potential inputs is `0` to `2^64 - 1`, or `2^64` total values. Assuming each value takes 1 nano-second to test, it would take approximately `584.5` years to exhaustively test just this single function.
+<!-- (2^64)/1000000000 / 60 / 60 / 24 / 365.25 = 584.54204609062639795168200370117 -->
+
+In contrast, a static verifier like Prusti is able to check the entire input space of a function with the help of the specifications of each function.
+
+When verification succeeds, you are guaranteed to not have a bug like a crash, overflow, or return value not fitting the specification.
+This assumes that you have manually verified any `#[trusted]` functions and have checked for correct termination of all functions.
+If the verification fails, you may have a bug, or your specifications are not strong enough.
+
+In other words: Testing can show the *presence* of bugs, verification can show the *absence* of bugs.
```

Comment:
I think this paragraph should have been added after the "The guarantees of testing…" one.